src/utils/Processing.ts

```diff
@@ -27,19 +27,34 @@ export function findInputDir(input: string): string {
     logger.info('path-to-fsh-defs defaulted to current working directory');
   }
 
+  // TODO: Legacy support. Remove when no longer supported.
+  // Use input/fsh/ subdirectory if not already specified and present
+  const inputFshSubdirectoryPath = path.join(input, 'input', 'fsh');
+  if (fs.existsSync(inputFshSubdirectoryPath)) {
+    input = path.join(input, 'input', 'fsh');
+  }
+
   // Use fsh/ subdirectory if not already specified and present
   const fshSubdirectoryPath = path.join(input, 'fsh');
-  if (fs.existsSync(fshSubdirectoryPath)) {
+  if (!fs.existsSync(inputFshSubdirectoryPath) && fs.existsSync(fshSubdirectoryPath)) {
     input = path.join(input, 'fsh');
-    logger.info('fsh/ subdirectory detected and add to input path');
+    logger.warn(
+      'SUSHI detected a "fsh" directory that will be used in the input path. ' +
+        'The top level "fsh" directory is being deprecated. FSH definitions in ' +
+        '"fsh" should be moved to "input/fsh".'
```

Comment:
Should we also say
* config.yaml moves to ${root}/sushi-config.yaml
* files in ig-data go in their respective folders starting from the root

??? (Only word it better?)

If so, maybe we could personalize the message w/ exact paths so it says something like:
* sushi-config.yaml moves to /Users/cmoesel/dev/MyIG/sushi-config.yaml
* ig-data/* files move to /Users/cmoesel/dev/MyIG/*
* .fsh files move to /Users/cmoesel/dev/MyIG/input/fsh/*

Reply:
I think this seems reasonable to update now. I like those last three bullets. Based off of #586 though, I think that sushi-config.yaml will go in `input/fsh` so I think the first bullet would be:
- sushi-config.yaml moves to /Users/cmoesel/dev/MyIG/input/fsh/sushi-config.yaml
I can update to this if you agree!

Reply:
Hmmm... I was thinking that w/ the new approach, `sushi-config.yaml` lives at the root, even if you have `input/fsh`.  To me, config files at the root just make more sense than in nested subfolders.  I think I assumed that @ngfreiter's PR was to rename the config, but that we'd address the new config location in a subsequent PR.

I guess that sort of challenges the notion, however, that SUSHI _switches_ the input folder to `input/fsh` when you run in a project w/ an `input/fsh` folder.  To me, I think I kind of like challenging that notion though.  It makes more sense to me if I think of it as SUSHI running in the root folder but scoping its input for `.fsh` files to `input/fsh`.  But of course, that probably makes a bunch more work, doesn't it?

Reply:
Well it turns out I have not waited for you to agree (sorry). But it is in df127fd (so there's still time to disagree). I'm not 100% sure about the wording, so let me know what you think of that and the location of sushi-config.

Reply:
Oh no! Your reply wasn't loaded when I replied! :panic:

Reply:
Wait, how does that make a bunch more work? I can reword the messages that say we "switch" the input folder to align with the idea of "scoping". I guess another question would be when do we switch support for the location of config.yaml? Was that part of this task?

Reply:
I assume it is more work because I also assume that SUSHI expects things to be in certain places relative to its working directory.  Maybe that is one (or two) too many assumptions.  Is it?